src/main/java/sg/ncl/MainController.java

```diff
@@ -1640,38 +1637,39 @@ public String checkApplyTeamInfo(
         teamFields.put("organisationType", teamPageApplyTeamForm.getTeamOrganizationType());
         teamFields.put("visibility", teamPageApplyTeamForm.getIsPublic());
 
+        String nclUserId = session.getAttribute("id").toString();
+
         HttpEntity<String> request = createHttpEntityWithBody(mainObject.toString());
-        ResponseEntity response = restTemplate.exchange(properties.getRegisterRequestToApplyTeam(session.getAttribute("id").toString()), HttpMethod.POST, request, String.class);
+        ResponseEntity response = restTemplate.exchange(properties.getRegisterRequestToApplyTeam(nclUserId), HttpMethod.POST, request, String.class);
 
         String responseBody = response.getBody().toString();
 
+
         try {
             if (RestUtil.isError(response.getStatusCode())) {
                 MyErrorResource error = objectMapper.readValue(responseBody, MyErrorResource.class);
                 ExceptionState exceptionState = ExceptionState.parseExceptionState(error.getError());
 
-                switch (exceptionState) {
-                    case APPLY_NEW_PROJECT_EXCEPTION:
-                        log.info("Apply new team fail at adapter deterlab");
-                        redirectAttributes.addFlashAttribute(MESSAGE, error.getMessage());
-                        break;
-                    case TEAM_NAME_ALREADY_EXISTS_EXCEPTION:
-                        log.info("Apply new team fail: team name already exists", teamPageApplyTeamForm.getTeamName());
-                        redirectAttributes.addFlashAttribute(MESSAGE, "Team name already exists.");
-                        break;
-                    default:
-                        log.info("Apply new team fail: registration service or adapter fail");
-                        // possible sio or adapter connection fail
-                        redirectAttributes.addFlashAttribute(MESSAGE, ERR_SERVER_OVERLOAD);
-                        break;
+                if (true) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this condition so that it does not always evaluate to "true" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2583)